### DISCUSSION TITLE
Reference code to fine-tune arabertv2 model for [MASK] prediction

### DISCUSSION THREAD

**Discussion by kwcckw** [aub-mind/arabert#116] [Q&A] [answered]
Hi,

Good day.

I would like to fine-tune arabertv2 with new dataset for [MASK] prediction purpose, could you point me to the correct sample?

Also is this sample relevant?
https://github.com/aub-mind/arabert/blob/master/examples/old/araBERT_(Updated_Demo_TF).ipynb

Please kindly advise.

Thank you.

**Comment by WissamAntoun** [comment, marked as answer]
for Masked Language modeling I suggest you use https://github.com/huggingface/transformers/blob/master/examples/pytorch/language-modeling/run_mlm.py script from huggingface. You can apply AraBERT preprocessing, before sending the training sentences to the script

**Reply by kwcckw** [reply]
Hi Wissam Antoun,

Thank you very much.
